pkg/ccl/backupccl/backup_test.go

```diff
@@ -8339,6 +8339,10 @@ func TestBackupOnlyPublicIndexes(t *testing.T) {
 	//  7. Drop index
 	//  8. Inc 4
 
+	// Disable declarative schema changer for this test, until the knobs are updated.
+	sqlDB.Exec(t, "SET use_declarative_schema_changer='off';")
+	sqlDB.Exec(t, "SET CLUSTER SETTING sql.defaults.use_declarative_schema_changer='off';")
```

Comment:
Educational: This is the first time I see this cluster setting. I've always known and only know the session setting. What is this cluster setting and why do we have it?

Reply:
It sets the default setting for the session variable so that any new connection will inherit the same value. This test uses multiple connections, so instead of setting this as disabled for each one, we can disable it across all of them.